db/migrate/20210327165829_add_average_rating_to_spaces.rb

```diff
@@ -0,0 +1,5 @@
+class AddAverageRatingToSpaces < ActiveRecord::Migration[6.0]
+  def change
+    add_column :spaces, :avg_rating, :float
```

Comment:
Ah so you did have to add a rating column after all? It couldn't be done on the fly?

Reply:
We opted to add a rating column, and when a review is created, the corresponding space's average rating is updated. 🤓